### PR TITLE
Mark use of deprecated type.

### PR DESCRIPTION
**Pull request by lrhn**
`NullThrownError` will be deprecated Real Soon Now (https://dart-review.googlesource.com/c/sdk/+/248901) since it isn't needed in sound null-safe code, and therefore should go away when support is dropped for unsound/pre null-safe code.

Missed this use in prior PR.

### CODE REVIEW

**Comment by lrhn** [comment]
@Hixie 
No tests should be needed. As soon as the deprecation lands in the Dart SDK (blocked by this CL), it will become a warning to remove the ignore.

**Review by Piinks — APPROVED** [review]
Flutter_LGTM
Reaching out to Hixie for test exemption.

**Review by Hixie — APPROVED** [review]
LGTM

**Pull request merged**
